### PR TITLE
2.0 release content

### PR DESCRIPTION
**Pull request by ashb**

### CODE REVIEW

**Comment by ashb** [comment]
Don't merge yet :)

**Comment by potiuk** [comment]
I feel tinging in the tips of my fingers :)

**Comment by mik-laj** [comment]
> Don't merge yet :)


Can change the base branch from master to airflow-2-0-docs?  See: https://github.com/apache/airflow-site/issues/314

**Review by turbaszek — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Comment by ashb** [comment]
> > Don't merge yet :)
> 
> If you're interested, then you can change the base branch from master to airflow-2-0-docs? See: #314

Should we merge that branch to master instead @mik-laj?

**Comment by mik-laj** [comment]
> Should we merge that branch to master instead @mik-laj?

When we release Airflow 2.0, we will be able to merge(or move head) the airflow-2-0-docs branch into the master. That way, you don't have to hold back on merging changes to apache/airflow.

 Besides, I have the impression that there may be a problem with the stability of the synchronization of changes, if a lot of changes will be merged into the master branch simultaneously.

**Comment by ashb** [comment]
Okay, so I'll retarget that branch, and then to publish it we just push the airflow-2-0-docs branch all as one go to master, right?

**Comment by mik-laj** [comment]
Yes. All in one push to limit file syncs.

**Comment by ashb** [comment]
In which case I'll merge this once I have verified the link to the astro blog (it's not published yet)

File: landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md
Comment (kaxil) [inline landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md:144]:
```suggestion
Thank you so much to all the contributors who got us to this point, in no particular order: Kaxil Naik, Daniel Imberman, Jarek Potiuk, Tomek Urbaszek, Kamil Breguła, Gerard Casas Saez, Xiaodong DENG, Kevin Yang, James Timmins, Yingbo Wang, Qian Yu, Ryan Hamilton and the 100s of others who keep making Airflow better for everyone.
```

File: landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md
Comment (kaxil) [inline landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md:142]:
```suggestion
We've tried to make as few breaking changes as possible and to provide deprecation path in the code, especially in the case of anything called in the DAG. That said,  please read throughUPDATING.md to check what might affect you. For example: We re-organized the layout of operators (they now all live under airflow.providers.*) but the old names should continue to work - you'll just notice a lot of DeprecationWarnings that need to be fixed up.
```

File: landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md
Comment (kaxil) [inline landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md:102]:
```suggestion
If you make heavy use of sensors in your Airflow cluster, you might find that sensor execution takes up a significant proportion of your cluster even with "reschedule" mode. To improve this,  we've added a new mode called "Smart Sensors".
```

File: landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md
Comment (kaxil) [inline landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md:98]:
```suggestion
We have also added an option to auto-refresh task states in Graph View so you no longer need to continuously press the refresh button :).

Check out [the screenshots in the docs](http://airflow.apache.org/docs/apache-airflow/stable/ui.html) for more.
```

**Review by kaxil — APPROVED** [review]

File: landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md
Comment (ashb) [reply landing-pages/site/content/en/blog/airflow-two-point-oh-is-here/index.md]: Can't apply this suggestion for some reason.

**Comment by ashb** [comment]
MONSTER PR now 😁

**Comment by ashb** [comment]
image

Ahahahahah :lolsob:

**Review by mik-laj — APPROVED** [review]

**Review by potiuk — APPROVED** [review]

**Pull request merged**
